src/deluge/model/settings/runtime_feature_settings.h

```diff
@@ -34,7 +34,7 @@ enum RuntimeFeatureStateToggle : uint32_t { Off = 0, On = 1 };
 // Declare additional enums for specific multi state settings (e.g. like RuntimeFeatureStateTrackLaunchStyle)
 enum RuntimeFeatureStateSyncScalingAction : uint32_t { SyncScaling = 0, Fill = 1 };
 
-/// Every setting needs to be delcared in here
+/// Every setting needs to be delcared in here - Warning - Probably don't use them in the rendering path
```

Comment:
The master comp is already used in the audio path, but it's just 1 check per render. I guess that's ok if we haven't noticed a performance change

Reply:
I checked all the calls to runtimeFeatureSettings.get and all references to RuntimeFeatureSettingType::MasterCompressorFx to make sure and didn't find an occurence that was in the audio path. If there is I can reconsider since the savings are about 5k